Sources/Property.swift

```diff
@@ -50,6 +50,39 @@ extension MutablePropertyProtocol {
 	}
 }
 
+extension PropertyProtocol {
+	/// Returns a `SignalProducer` that sends a new value each time this
+	/// property's value changes, using the supplied `transform` to supply the
+	/// value being sent.
```

Comment:
This reads like documentation for `map`, so it could use some cleanup.

Of note:

1. `that sends a new value each time this property's value changes` isn't true. The point of the operator is that it doesn't guarantee a 1-to-1 mapping of input to output.

2. It doesn't communicate what makes this lazy.

It looks like the other instances of `lazyMap` have better documentation, so you may just need to copy that here.